src/components/Basket.js

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+
+class Basket extends React.Component {
+  constructor() {
+    super();
+
+    this.handleClick = this.handleClick.bind(this);
+    this.getBasket = this.getBasket.bind(this);
+  }
+
+  handleClick(event) {
+    event.preventDefault();
+    fetch(`/order`, {
+      method: "post",
+      body: JSON.stringify(this.props.currentOrder),
+      headers: {
+        "Content-Type": "application/json"
+      }
+    })
+      .then(function(response) {
+        return response.json();
+      })
+      .catch(error => console.log(error));
+
+    this.props.receiveWipeOrder();
+    alert("Thank you for your order!");
+  }
+
+  getBasket(total) {
+    const deliveryCharge = total * 0.1;
```

Comment:
It would be slightly cleaner to implement this function as a separate child component and make this component simpler. Also, personal opinion` an if/else would be easier to read than multiline ternary